modules/core/src/main/java/org/apache/ignite/IgniteCache.java

```diff
@@ -145,8 +147,10 @@
      *  <li>for transactional caches:
      *  <p>values across the topology will be replaced by latest versioned value:
      *  <ul>
-     *      <li>automaticaly for OPTIMISTIC || READ_COMMITTED transactions</li>
-     *      <li>at commit() for PESSIMISTIC && !READ_COMMITTED transactions</li>
+     *      <li>automatically for transactions that have {@link TransactionConcurrency#OPTIMISTIC} concurrency mode
+     *          or {@link TransactionIsolation#READ_COMMITTED} isolation level</li>
+     *      <li>at commit() phase for transactions that have {@link TransactionConcurrency#PESSIMISTIC} concurrency mode
+     *          and isolation level other than {@link TransactionIsolation#READ_COMMITTED}</li>
```

Comment:
not sure that "other than" is a correct English phrase.

Reply:
I am definitely not an expert, but I think it is absolutely correct.
It is widely used in the Java API, for instance:
```java
An IllegalAccessException is thrown when an application tries to reflectively create an instance (other than an array), set or get a field, or invoke a method, but the currently executing method does not have access to the definition of the specified class, field, method or constructor.
```

Also, the definition of 'other than' can be found at Cambridge Dictionary, Merriam-Webster and etc.
https://dictionary.cambridge.org/dictionary/english/other-than
https://www.merriam-webster.com/dictionary/other%20than